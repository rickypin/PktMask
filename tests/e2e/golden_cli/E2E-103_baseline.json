{
  "test_id": "E2E-103",
  "name": "TLS 1.3 Protocol",
  "input_file": "tests/data/tls/tls_1_3_0-RTT-2_22_23_mix.pcap",
  "input_hash": "7f0eb211ffb44bb94d886f25a31e30f98079ac085efa3bb1482de09c7a59ffdb",
  "cli_args": [
    "--dedup",
    "--anon",
    "--mask"
  ],
  "output_hash": "afac65f2d708d572f37809e5a7d8a30125cbde215d9554e8092f8d78b16d38f0",
  "timestamp": "2025-10-09T21:31:16.818679",
  "version": "1.0-cli",
  "note": "Generated using CLI only - completely decoupled from API"
}