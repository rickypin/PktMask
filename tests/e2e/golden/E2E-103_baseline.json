{
  "test_id": "E2E-103",
  "name": "TLS 1.3 with 0-RTT",
  "input_file": "tests/data/tls/tls_1_3_0-RTT-2_22_23_mix.pcap",
  "input_hash": "7f0eb211ffb44bb94d886f25a31e30f98079ac085efa3bb1482de09c7a59ffdb",
  "config": {
    "dedup": true,
    "anon": true,
    "mask": true
  },
  "output_hash": "afac65f2d708d572f37809e5a7d8a30125cbde215d9554e8092f8d78b16d38f0",
  "output_file_size": 5846,
  "packet_count": 31,
  "stats": {
    "packets_processed": 31,
    "packets_modified": 41,
    "duration_ms": 542.4561500549316,
    "stages": [
      {
        "name": "DeduplicationStage",
        "packets_processed": 31,
        "packets_modified": 0,
        "duration_ms": 5.511760711669922
      },
      {
        "name": "AnonymizationStage",
        "packets_processed": 31,
        "packets_modified": 31,
        "duration_ms": 10.671854019165039
      },
      {
        "name": "Mask Payloads",
        "packets_processed": 31,
        "packets_modified": 10,
        "duration_ms": 13.308048248291016
      }
    ]
  },
  "timestamp": "2025-10-09T19:39:50.337074",
  "version": "1.0"
}